StreamChatClient_v3/ChatClient.swift

```diff
@@ -0,0 +1,85 @@
+//
+// ChatClient.swift
+// Copyright © 2020 Stream.io Inc. All rights reserved.
+//
+
+import CoreData
+import Foundation
+
+public final class ChatClient {
```

Comment:
This is in the separate target, you can use just `Client`.

Reply:
I disagree, `Client` is too general, IMHO it makes sense to have `ChatClient`

Reply:
Swift has namespaces. Especially for that. Please let's use it and our SDK will look more solid.

Reply:
https://exceptionnotfound.net/bikeshedding-the-daily-software-anti-pattern/

Reply:
We're 3, we can always apply democracy :)